.travis.yml

```diff
@@ -47,7 +48,7 @@ script:
   - ./bessctl/bessctl -- daemon start -- run testing/run_module_tests
 
 after_success:
-  - "[[ ${DEBUG:-0} == 0 ]] || bash <(curl -s https://codecov.io/bash)"
+  - "[[ ${DEBUG:-0} == 0 ]] || (bash <(curl -s https://codecov.io/bash) && cd core && ./coverage run)"
```

Comment:
hmmm... I think the `cd` has no side effect outside of its subshell. But yeah, I'll make it clearer.